text/0141-linking-traces.md

```diff
@@ -0,0 +1,447 @@
+- [Logrocket sessions](https://docs.logrocket.com/docs/what-defines-a-session) can span multiple tabs and have a lifetime of 30 minutes, with some further heuristics for closing tabs
+
+Again, linked traces are not identical to sessions but they can also help answer questions that sessions could answer.
+
+## Implications of Long-lived traces
+
+As mentioned [above](#frontend-traces-and-user-journeys), our current long-lived traces in Browser JS have some negative implications on the Sentry UX:
+
+- Multiple root spans (transactions): Our product can (and should) handle multiple root spans. However, there are some rough edges which are hard to solve from a UI/UX perspective. Our Trace explorer gives traces a name, for instance. If multiple root spans exist, we can only use a heuristic for which span name to use for the "trace name". Furthermore, multiple root spans contradict the conventional concept of a trace, where a trace should only consist of one root span.
+- Trace Duration: As soon as a trace spans more than one root span or multiple events, the duration of the trace itself becomes meaningless. While we strongly recommend to generally avoid deducing information from the trace duration, we could improve the current situation by sending less long-lived traces
+- Context: While long lived traces do potentially provide more context, given they could include spans or events that happened before an error, they can sometimes contain "too much" context. For instance, very long-running traces in web applications without navigations would contain close to the entire user journey. This makes it harder to understand cause-effect relationships within a trace.
+- UI/UX of the trace view: Long running traces are not ideally displayed in the trace view, given the x-axis would span a large time frame. This makes it necessary for users to zoom in and pan around much more than for single-root span traces.
+- Sampling and quota increase: If a trace is sampled positively, all transactions within this trace will be sampled positively. This also means that while the positively sampled trace is ongoing, we'd always propagate the positive sampling decision via the `sentry-trace` header to downstream services on outgoing requests. Assuming these services are configured to only sample already sampled traces, we'd send more events on an org-wide basis than if we rolled the dice more often (like in short-lived traces). This [increases quota usage](https://github.com/getsentry/sentry-ruby/issues/2318) for our customers without them necessarily being aware of it. 
+ 
+If we can link traces, we can ultimately shorten individual trace lengths and remove most of these disadvantages again.
+
+To be clear, the Sentry product will always need to support long-lived/multiple root span traces. While we can remove them from SDKs supporting this model, older SDK versions will stick around and users can also manually create long-lived traces.
+
+# Options Considered
```

Comment:
i'm missing somewhere here a short bullet list of titles of options considered. A lot to digest, and wasn't entirely clear from ## vs ### markdown which were options vs sub features like setting previous trace

```suggestion
# Options Considered

1. [Preferred] [Span Links](#preferred-span-links)
2. [Previous Trace Id](#alternative-previous-trace-id)
```